src/features/override-emotes.js

```diff
@@ -32,7 +32,7 @@ module.exports = function() {
                         } else if ($emote.data('channel') && $emote.data('channel') === 'BetterTTV Emotes') {
                             return 'Emote: ' + raw + '<br />BetterTTV Emoticon';
                         } else if ($emote.data('channel')) {
-                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel');
+                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel') + ' (BTTV)';
```

Comment:
Yeah, I'm wasn't 100% sure about this, I'm not 100% sure how up to date and accurate that list is kept.

The other two options I can think of are:
1. Set a data attribute on BTTV emotes
2. look at the href and check for BTTV cdn